fundingmanager.go

```diff
@@ -815,7 +831,7 @@ func (f *fundingManager) handleFundingOpen(fmsg *fundingOpenMsg) {
 	// party is attempting to dictate for our commitment transaction.
 	err = reservation.CommitConstraints(
 		uint16(msg.CsvDelay), msg.MaxAcceptedHTLCs,
-		msg.MaxValueInFlight, msg.ChannelReserve,
+		msg.MaxValueInFlight, msg.HtlcMinimum, msg.ChannelReserve,
```

Comment:
The `HtlcMinimum` value they sent is _their_ min value, now what they're dictating for us. The other three values are prescribing constraints for the remote node, while `HtlcMinimum` is them telling _us_ their min value.

Reply:
If you look at the `ChannelConstraints' I changed this around, such that all the constraints found in a node's `LocalChanCfg` is the constraints _they_ must _not violate_. I found this to be a bit more logical way of ordering the constraints, and makes it such that a node can validate an update against it's local constraints when sending, and against the remote constraints when receiving.

If my understanding of the spec is correct, this means that for all the committed constraints here:  a node must always check that these are not violated when _adding_ an HTLC (`channel.go:AddHTLC()`). 

When receiving a new signature from the remote, it checks that the remote constraints are not violated. 

I found this way of organising constraints less confusing. Will it work?

Reply:
Ahh, I see. That makes sense.